src/content/glossary/index.md

```diff
@@ -233,6 +241,14 @@ The ENS registry is a single central [contract](#smart-contract) that provides a
 
 In the context of cryptography, lack of predictability or level of randomness. When generating secret information, such as [private keys](#private-key), algorithms usually rely on a source of high entropy to ensure the output is unpredictable.
 
+### execution client {#execution-client}
+
+After the merge to [proof-of-stake](#pos), existing Ethereum Mainnet clients will continue to host the [Ethereum Virtual Machine](#evm), validate transactions and execute state transitions but will not participate in consensus. These clients will therefore be referred to as "execution clients".
```

Comment:
I think this one isn't 100% accurate. My suggestion:

> Execution clients (f.k.a. "Eth1 clients"), such as Besu, Erigon, go-ethereum, Nethermind, are tasked with processing and broadcasting transactions, as well as with managing Ethereum's state. They run the computations for each transaction in the Ethereum Virtual Machine to ensure that the rules of the protocol are followed. Today, they also handle proof of work consensus. After the transition to proof of stake, they will delegate this to consensus clients.

Reply:
Thanks a lot for improving this @timbeiko!